src/Services/publicApiService.js

```diff
@@ -2,12 +2,19 @@ import axios from 'axios';
 
 const config = {
     headers: {
-        Group: 01                //Aqui va el ID del equipo!!
+        Group: 2
     }
 }
 
-const Get = () => {
-    axios.get('https://jsonplaceholder.typicode.com/users', config)
+const Get = (endpoint, id = '') => {
+
+    const isUndefinedEndpoint = endpoint == null;
```

Comment:
Esta linea de codigo no tiene mucho sentido. Seria mejor que el if de la linea 13 lo escribas asi: 

`if(!endpoint){
        throw new Error("parameter 'endpoint' is not defined.");
    }`